pynest/examples/intrinsic_currents_spiking.py

```diff
@@ -134,8 +134,7 @@
 # a dictionary with entry ``times`` containing timestamps for all
```

Comment:
Above: Remove Python-level loop for `Connect()`.

Reply:
We looked at this from the connection generation point of view, and were not sure how this could be improved with the currently available syntax. `receptor_type` does not take an array argument at the moment. This has to be discussed again in the context of the connection generation syntax.